client/src/components/util/InputForm.tsx

```diff
@@ -0,0 +1,65 @@
+import { Flex, Input, Text } from '@chakra-ui/react';
+import { ChangeEvent } from 'react';
+import { DEFAULT_INPUT_THEME, FontBorderTheme } from '../../constants/theme';
+
+interface InputProps {
+  name: string;
+  value: string;
+  onChange?: (e: ChangeEvent<HTMLInputElement>) => void;
+  type?: string;
+  w?: string;
+  h?: string;
+  placeholder?: string;
+  errorMessage?: string;
+  regex: RegExp;
+  theme: FontBorderTheme;
+}
```

Comment:
optional parameter이 젤 밑에 오도록!

```suggestion
interface InputProps {
  name: string;
  value: string;
  regex: RegExp;
  theme: FontBorderTheme;
  onChange?: (e: ChangeEvent<HTMLInputElement>) => void;
  type?: string;
  w?: string;
  h?: string;
  placeholder?: string;
  errorMessage?: string;
}
```

Reply:
이건 제가 ESLint 설정에서 만질 수 있는지 찾아볼게요!